src/js/binaryen.js-post.js

```diff
@@ -450,32 +450,25 @@ function wrapModule(module, self) {
 
   self['local'] = {
     'get': function(index, type) {
-      return Module['_BinaryenGetLocal'](module, index, type);
+      return Module['_BinaryenLocalGet'](module, index, type);
     },
     'set': function(index, value) {
-      return Module['_BinaryenSetLocal'](module, index, value);
+      return Module['_BinaryenLocalSet'](module, index, value);
     },
     'tee': function(index, value) {
-      return Module['_BinaryenTeeLocal'](module, index, value);
+      return Module['_BinaryenLocalTee'](module, index, value);
     }
   }
 
-  self['getLocal'] = self['local']['get'];
-  self['setLocal'] = self['local']['set'];
-  self['teeLocal'] = self['local']['tee'];
-
   self['global'] = {
     'get': function(name, type) {
-      return Module['_BinaryenGetGlobal'](module, strToStack(name), type);
+      return Module['_BinaryenGlobalGet'](module, strToStack(name), type);
     },
     'set': function(name, value) {
-      return Module['_BinaryenSetGlobal'](module, strToStack(name), value);
+      return Module['_BinaryenGlobalSet'](module, strToStack(name), value);
     }
   }
 
-  self['getGlobal'] = self['global']['get'];
-  self['setGlobal'] = self['global']['set'];
-
   self['currentMemory'] = self['current_memory'] = function() {
```

Comment:
Just noticed:
* `current_memory` -> `memory.size` / `CurrentMemory` -> `MemorySize`?
* `grow_memory` -> `memory.grow` / `GrowMemory` -> `MemoryGrow` ?

Reply:
hmm, yeah, would be nice to make these consistent as well, eventually.

Reply:
Good find! I'll rename all that as well in code and merge those methods into `memory` category in binaryen.js.